app/src/main/res/values-de/strings.xml

```diff
@@ -8,7 +8,7 @@
     <string name="main_activity_hint">Eine Benachrichtigung wird angezeigt, wenn der Klingelton stummgeschalten ist und Medien über lokale Lautsprecher abgespielt werden würden.</string>
     <string name="report_issues_github">Fehler auf GitHub melden</string>
     <string name="error_no_browser_found">Kein Browser gefunden</string>
-    <string name="version">Version %s</string>
+    <string name="app_version">Version %s</string>
```

Comment:
Can you revert the changes in `values-*/strings.xml`? These files are managed by Crowdin.

Reply:
I'll remove the string as it is not needed anymore on the app